miniwob/tests/test_api.py

```diff
@@ -1,25 +1,30 @@
 """Test integration with Gymnasium API."""
 import gymnasium
+import pytest
 from gymnasium import spaces
 from gymnasium.utils.env_checker import check_env
 from gymnasium.wrappers.flatten_observation import FlattenObservation
 
 import miniwob  # noqa: F401
+from miniwob.tests.utils import ALL_TESTABLE_MINIWOB_ENVS
 
 
 class TestGymAPI:
```

Comment:
This looks completely fine, Im just more use to functional style testing rather than class based testing. 
Is `@pytest.fixture(params=...)` equal to `@pytest.mark.parameterize("env_id", ALL_...)`?

Reply:
Yes, it should do the same thing.